src/analysis/utils/cost-utils.ts

```diff
@@ -213,7 +211,7 @@ export const getGalaxyComputeCost = (app: App): number => {
  */
 export const getGalaxyDiskCost = (
   disk:
-    | DecoratedPersistentDisk
+    | PersistentDisk
```

Comment:
nit: this could also be a `Pick<`, an even shorter one

Reply:
Also, I realized I don't know what `key: | Type` means - does the initial ` | ` do something? Or is it just sugar to make the types look good on multiple lines?